sound/soc/sof/sof-client.c

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
This would be the only file with a different license in soc/sof/ Not 100% sure about this.

Reply:
@kv2019i this is correct. We cannot export GPL only APIs and make the file dual licensed.